src/integer/mat_z/get.rs

```diff
@@ -142,18 +145,13 @@ mod test_get_entry {
     #[test]
     fn big_negative() {
         let mut matrix = MatZ::new(5, 10).unwrap();
-        let mut value = "-".to_string();
-        value.push_str(&"1".repeat(65));
         matrix
-            .set_entry(1, 1, Z::from_str(&value).unwrap())
+            .set_entry(1, 1, Z::from_str(&format!("-{}", u64::MAX)).unwrap())
             .unwrap();
 
         let entry = matrix.get_entry(1, 1).unwrap();
 
-        let mut test_entry = "-".to_string();
-        test_entry.push_str(&"1".repeat(65));
-
-        assert_eq!(Z::from_str(&test_entry).unwrap(), entry);
+        assert_eq!(Z::from_str(&format!("-{}", u64::MAX)).unwrap(), entry);
     }
```

Comment:
Using a variable to store the string like `value_str` makes it more clear that the strings are the same. 

Also not fitting into i64 does not really change the behavior compared to the previous test case. 
However, not fitting into 64 bit could in theory. I personally would also be fine without this test case, since the different behavior when flint uses pointers is already covered by the previous test.
```rust
    /// Ensure that getting entries works with large negative numbers (not fitting into 64 bit).
    #[test]
    fn big_negative() {
        let mut matrix = MatZ::new(5, 10).unwrap();
        let value_str = &format!("-{}", u64::MAX); // value is not fitting into 64 bit
        matrix
            .set_entry(1, 1, Z::from_str(value_str).unwrap())
            .unwrap();

        let entry = matrix.get_entry(1, 1).unwrap();

        assert_eq!(Z::from_str(value_str).unwrap(), entry);
    } 
```
(could not make this as a code suggestion since the changed lines include deletions)